projects/sandbox/datagen/datagen/scripts/timeslide_waveforms.py

```diff
@@ -47,7 +47,60 @@ def main(
     verbose: bool = False,
 ):
     """
-    Generates the waveforms for a single segment
+    Generates the waveforms for a single segment.
+
+    Args:
+        start:
+            GPS time of the beginning of the testing dataset
+        stop:
+            GPS time of the end of the testing dataset
+        shifts:
+            A list of shifts in seconds. Each value corresponds to the
+            the length of time by which an interferometer's timeseries is
+            moved during one shift. For example, if `ifos = ["H1", "L1"]`
+            and `shifts = [0, 1]`, then the Livingston timeseries will be
+            advanced by one second per shift, and Hanford won't be shifted
+        background:
```

Comment:
We should probably make this `background_dir` to make it clear what it is. The only constraint on the files is that they ought to be readable by `TimeSeries.read` as well I think, right?

Reply:
Ok looking at `load_psds` now and realizing we do the load manually for some reason. I would say we should just update this to use `TimeSeries.read` but again that's probably out of scope so your description of the constraints is probably fair for now.